contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -51,13 +51,14 @@ type devCfg struct {
 	balancesFile    string
 
 	// Node Configuration
-	minimal    bool
-	verbose    bool
-	noWatch    bool
-	noReplay   bool
-	maxGas     int64
-	chainId    string
-	serverMode bool
+	minimal       bool
+	verbose       bool
+	noWatch       bool
+	noReplay      bool
+	maxGas        int64
+	chainId       string
+	serverMode    bool
+	webHelpRemote string
```

Comment:
Embed `gnoweb.Config` instead of using individual gnoweb flags.

Reply:
`gnodev.Config` should include specific flags and embed configurations like `gnoweb.Config`, `gnoland.Config`, etc.

Reply:
Added a commit to embed gnoweb.Config